cncf-transition/versioning.md

```diff
@@ -0,0 +1,18 @@
+# Kubeflow Versioning Policy
+
+[Kubeflow Working Groups](https://github.com/kubeflow/community/blob/master/wg-list.md) (WG) follows
+ [semantic versioning](https://semver.org/) terminology expressed as X.Y.Z format, where X is the major version,
+ Y is the minor version, and Z is the patch version.
+
+For pre-release artifacts, `rc` and a number incrementing from 0 are appended to the end of the version to indicate
+ that it is a pre-release. For example, `X.Y.Z-rc.0`.
+
+In addition, some Kubeflow WGs leverage feature stages `alpha` and `beta` before releasing a stable version.
+ For example, `X.Y.Z-alpha.0`.
+
+For more details, see specification of each working group version policy and release process
+- [Katib](https://github.com/kubeflow/katib/tree/master/docs/release#release-process)
+- Manifests
+- Notebooks
```

Comment:
Are there policies for these as well?

Reply:
I couldn't find any release process / version policies documented for these two WGs. Hoping @kimwnasptd can chime in if there is one available.

I decided to still list them here without a link to indicate that there is no specific policy set for these two WGs.

Reply:
It looks like this list is for the ones with specific process though.

Reply:
Should we file issues to Manifest and Notebooks WG to document their versioning policy?

Reply:
Got it - I can understand the confusion. I'll go ahead and remove Manifest and Notebooks from the list for now to address @terrytangyuan 's concern and also file the issues in Manifest and Notebook to ensure their process is documented which addresses @james-jwu's comment. 

Once the docs are available for Manifest and Notebok, we can add them back to this list